website/docs/API.md

```diff
@@ -336,6 +336,10 @@ function waitFor<T>(
 
 Waits for non-deterministic periods of time until your element appears or times out. `waitFor` periodically calls `expectation` every `interval` milliseconds to determine whether the element appeared or not.
 
+:::info
+In order to properly use `waitFor` you need at least React >=16.9.0 (featuing async `act`) or React Native >=0.60 (which comes with React >=16.9.0).
```

Comment:
I'm not sure if "In order to properly use" is good-enough wording. Maybe: "To avoid `act` warnings"?
```suggestion
To avoid `act` warnings when using `waitFor` you need React >=16.9.0 (featuring async `act`) or React Native >=0.60 (which comes with React >=16.9.0).
```
Not sure which is better, feel free to pick.

Reply:
It's not about warning only, async tests will actually not work properly if you don't have R >= 16.9.